catalog/dags/providers/provider_api_scripts/cleveland_museum.py

```diff
@@ -69,14 +71,14 @@ def get_record_data(self, data):
         }
 
     @staticmethod
-    def _get_image_data(image_data):
+    def _get_image_data(image_data) -> dict | None:
         # Returns the best available image in the `image_data` dict,
         # preferring `web` and falling back to other types.
-        if image_data is None:
+        if not image_data:
             return None
 
         for key in ["web", "print", "full"]:
-            if keyed_image := image_data.get(key):
+            if (keyed_image := image_data.get(key)) and "url" in keyed_image:
```

Comment:
Ooo, good call moving this check upstream! Do you mind adding a comment on why we discard records without `"url"` here?